docs/source/commands/dev.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: The Rover dev command
+description: Query multiple GraphQL APIs from localhost
+---
+
+`rover dev` starts a local router that can query across one or more running GraphQL APIs (subgraphs)
+through one endpoint (supergraph). As you add, edit, and remove subgraphs, `rover dev` automatically
+composes all of their schemas into a new supergraph schema, and the router reloads. 
+
+Think plug-n-play USB devices but with your GraphQL APIs!
+
+⚠️ Do not run this command in production! ⚠️ It is intended for local development.
+
+## Starting the first `rover dev` session
+
+Before starting `rover dev`, you need to start your GraphQL API (subgraph). Note the URL it is running on, and the location of the schema on the filesystem if it exists. If you don't have a local schema file, introspection will need to be enabled on the subgraph.
+
+An example `rover dev` invocation might look something like this:
+
+`rover dev --name products --schema ./products.graphql --url http://localhost:4000`
+
+This will start the main `rover dev` session, watching `./products.graphql` for changes to the schema, and start up a local Apollo Router that can be queried at http://localhost:3000. This session will start listening for other `rover dev` sessions that include other subgraph schemas.
+
+### Stopping the first `rover dev` session
+
+If you stop the first `rover dev` session (by pressing ctrl+c), it will shut down the router and any attached `rover dev` sessions will be detached and shut down.
+
+## Attaching more subgraphs to the main `rover dev` session
+
+For each new subgraph, run `rover dev` in a new terminal with the necessary arguments that refer to your GraphQL API (subgraph). If the `--supergraph-port` argument is equivalent to a running `rover dev` session, the new session will attach to the running session and provide it with updates to the subgraph schema. The main session will then re-compose the supergraph schema, printing the build result in the output of the main session and the attached session.
```

Comment:
Can we take out the `--supergraph-port` argument part here? Maybe move it to a new section? 

Also, how does this style tweak feel:
```suggestion
To add a new subgraph to a session, run `rover dev` in a new terminal with the necessary arguments for your GraphQL API (i.e., subgraph). If a session is already running, Rover automatically attaches subsequent `rover dev` processes to that existing session. 

When you add a new subgraph to a session, Rover handles recomposing the supergraph schema and printing the build result in the outputs of both the main session and the attached session.
```